### ISSUE TITLE
Compiler error on on Ubuntu 20.04: ITEM_MAGICWALL_NOPVP

### ISSUE THREAD

**Issue by Znote** [nekiro/TFS-1.4-Downgrades#12]
**Protocol version**
8.60

**Describe the bug**
A compiler error prevents me from building latest source on Ubuntu 20.04

**To Reproduce**
This tutorial, but targeting the repository this issue resides in. 
https://github.com/otland/forgottenserver/wiki/Compiling-on-Ubuntu

**Expected behavior**
A cleanly compiled linux binary file.

**Screenshots**
![image](https://user-images.githubusercontent.com/1552144/117572736-81131980-b0d4-11eb-8961-9dec0cbeaf8b.png)

**Additional context**
-- The C compiler identification is GNU 9.3.0
```
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Found PugiXML: /usr/include
-- Found LuaJIT: /usr/lib/x86_64-linux-gnu/libluajit-5.1.so;/usr/lib/x86_64-linux-gnu/libm.so (found version "2.1.0-beta3")
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake (found suitable version "1.71.0", minimum required is "1.66.0") found components: date_time system filesystem iostreams
-- IPO / LTO enabled
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ubuntu/server/github/build
ubuntu@dev:~/server/github/build$ make -j 3
Scanning dependencies of target check_git
[  1%] Checking the git repository for changes...
[  1%] Built target check_git
Scanning dependencies of target tfs
[  2%] Building CXX object CMakeFiles/tfs.dir/cmake_pch.hxx.gch
[  3%] Building CXX object CMakeFiles/tfs.dir/src/otpch.cpp.o
[  5%] Building CXX object CMakeFiles/tfs.dir/src/ban.cpp.o
[  6%] Building CXX object CMakeFiles/tfs.dir/src/actions.cpp.o
[  7%] Building CXX object CMakeFiles/tfs.dir/src/baseevents.cpp.o
[  9%] Building CXX object CMakeFiles/tfs.dir/src/bed.cpp.o
[ 10%] Building CXX object CMakeFiles/tfs.dir/src/chat.cpp.o
[ 11%] Building CXX object CMakeFiles/tfs.dir/src/combat.cpp.o
[ 13%] Building CXX object CMakeFiles/tfs.dir/src/condition.cpp.o
/home/ubuntu/server/github/src/combat.cpp: In static member function ‘static void Combat::combatTileEffects(const SpectatorVec&, Creature*, Tile*, const CombatParams&)’:
/home/ubuntu/server/github/src/combat.cpp:587:16: error: ‘ITEM_MAGICWALL_NOPVP’ was not declared in this scope; did you mean ‘ITEM_MAGICWALL_SAFE’?
  587 |       itemId = ITEM_MAGICWALL_NOPVP;
      |                ^~~~~~~~~~~~~~~~~~~~
      |                ITEM_MAGICWALL_SAFE
/home/ubuntu/server/github/src/combat.cpp:589:16: error: ‘ITEM_WILDGROWTH_NOPVP’ was not declared in this scope; did you mean ‘ITEM_WILDGROWTH_SAFE’?
  589 |       itemId = ITEM_WILDGROWTH_NOPVP;
      |                ^~~~~~~~~~~~~~~~~~~~~
      |                ITEM_WILDGROWTH_SAFE
/home/ubuntu/server/github/src/combat.cpp: In member function ‘void MagicField::onStepInField(Creature*)’:
/home/ubuntu/server/github/src/combat.cpp:1415:12: error: ‘ITEM_MAGICWALL_NOPVP’ was not declared in this scope; did you mean ‘ITEM_MAGICWALL_SAFE’?
 1415 |  if (id == ITEM_MAGICWALL_NOPVP || id == ITEM_WILDGROWTH_NOPVP) {
      |            ^~~~~~~~~~~~~~~~~~~~
      |            ITEM_MAGICWALL_SAFE
/home/ubuntu/server/github/src/combat.cpp:1415:42: error: ‘ITEM_WILDGROWTH_NOPVP’ was not declared in this scope; did you mean ‘ITEM_WILDGROWTH_SAFE’?
 1415 |  if (id == ITEM_MAGICWALL_NOPVP || id == ITEM_WILDGROWTH_NOPVP) {
      |                                          ^~~~~~~~~~~~~~~~~~~~~
      |                                          ITEM_WILDGROWTH_SAFE
[ 14%] Building CXX object CMakeFiles/tfs.dir/src/configmanager.cpp.o
make[2]: *** [CMakeFiles/tfs.dir/build.make:169: CMakeFiles/tfs.dir/src/combat.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:125: CMakeFiles/tfs.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

**Comment by nekiro** [comment]
You can apply #8 atm to fix this, but I requested changes and it will be fixed after the aforementioned pr will be merged.

**Comment by nekiro** [comment]
Fixed in #8
